easybuild/easyconfigs/m/Markdown/Markdown-3.6-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
+
+name = 'Markdown'
+version = '3.6'
+
+homepage = 'https://python-markdown.github.io/'
+description = """This is a Python implementation of John Gruber's Markdown.
+It is almost completely compliant with the reference implementation, though there are a few known issues.
+Additional features are supported by the Available Extensions.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['ed4f41f6daecbeeb96e576ce414c41d2d876daa9a16cb35fa8ed8c2ddfad0224']
+
```

Comment:
```suggestion

builddependencies = [
    ('binutils', '2.40'),
]

```

Reply:
IMO Such things should be on a single line to reduce noise and IIRC we do that in most other places. I.e.:
`builddependencies = [('binutils', '2.40')]`